common-utils/misc-utils.js

```diff
@@ -44,4 +44,46 @@ export class MiscUtils {
     this.testRunner.get('[data-test-subj="queryInput"]').type('{selectall}{backspace}')
     this.testRunner.get('[data-test-subj="querySubmitButton"]').click()
   }
+
+  /**
+   * Add sample data of Sample eCommerce orders, Sample flight data, Sample web logs to dashboard
+   */
+  addSampleData() {
+    this.testRunner.visit('app/home#/tutorial_directory')
+    this.testRunner.get('button[data-test-subj="addSampleDataSetecommerce"]').should('be.visible').click()
```

Comment:
I would prefer to extract there `'button[data-test-subj="addSampleDataSetecommerce"]'` selector/xpath to a const for readability and maintainability